### PR TITLE
Fixing and reenabling ServiceBus templates

### PR DESCRIPTION
**Pull request by mathewc**

### CODE REVIEW

File: Templates/ServiceBusQueueTrigger-NodeJS/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+﻿module.exports = function(context, myQueueItem) {
+    context.log('Node.js ServiceBus queue trigger function processed message', myQueueItem);
+    context.done(null, OutPutQueueItem);
```
Comment (soninaren) [inline Templates/ServiceBusQueueTrigger-NodeJS/index.js]: This should be `context.done()` since there is not output binding

Comment (mathewc) [inline Templates/ServiceBusQueueTrigger-NodeJS/index.js]: Good catch - I'm testing it in prod and already caught that. I'll make sure it runs e2e.

Comment (mathewc) [inline Templates/ServiceBusQueueTrigger-NodeJS/index.js]: Fixed.

**Comment by soninaren** [comment]
:+1:  Looks good

**Pull request merged**